src/assets/css/filesystem.css

```diff
@@ -149,7 +149,7 @@ section#filesystem.list-view > div#fs_disp_container:not(.disks) > div > h4:nth-
 section#filesystem.list-view > div#fs_disp_container:not(.disks) > div > h4:nth-of-type(3) { width: 38%; }
 
 div#fs_disp_container.disks {
-    display: flex;
+    padding: 0 0.5vw;
```

Comment:
Check line :109 of the same file, when list view (`Ctrl+Shift+L`) is enabled this will break.

Perhaps using flex-wrap instead of deactivating flexbox would be easier, and might also retain the center alignment of the disk view items? :thinking:

Reply:
Ah, sorry about that; wasn't aware of list view :sweat_smile: 
Actually, simply adding in flex-wrap brings the list view for disks to be pretty much the same as the normal grid view (the only slight difference being spacing between items due to different grid-gap attributes).
Here's what both views look like on my screen:

Grid View:
![grid_view](https://user-images.githubusercontent.com/37730685/98448623-637bec00-2153-11eb-8fa9-fb7784b411a3.png)

List View:
![list_view](https://user-images.githubusercontent.com/37730685/98448627-6aa2fa00-2153-11eb-91ed-ea20ad40c05e.png)

Is that alright or is it preferred to have horizontal scrolling on the original disk view instead?

Reply:
No, it's good! Can you scroll down if there are multiple screens of disks though? Just want to make sure

Reply:
Yep, I'm able to scroll after copy pasting a couple of divs in dev tools.